dapps/assets/src/components/digitaltwins/DigitalTwins.vue

```diff
@@ -52,32 +56,50 @@ import DigitalTwinsComponent from './DigitalTwins';
 export default DigitalTwinsComponent;
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
We should never define any global styling within a component.  Please move evan-table to global ui-libs. Also `evan-table-body-row ` and `evan-table-head` can be removed. `evan-table` should be enough to style everything with one class.

```suggestion
<style lang="scss" scoped>
```

Reply:
Yes, this is a quick solution to share the evan-table style for both tables. This will be done in a refactor ticket (https://evannetwork.atlassian.net/browse/CORE-791)